flashcardschemas.py

```diff
@@ -1,6 +1,7 @@
 from collections import namedtuple
 
-ChineseRecognitionOnly = namedtuple('ChineseRecognitionOnly', 'simplified traditional pinyin zhuyin definition extra_definition words_with_same_pronunciation words_with_same_definition tags')
+ChineseRecognitionOnly = namedtuple('ChineseRecognitionOnly', 'simplified traditional pinyin zhuyin simplified_stroke traditional_stroke definition extra_definition words_with_same_pronunciation words_with_same_definition tags')
```

Comment:
please make a new schema - `ChineseRecognitionOnlyWithStroke` or something along those lines

Reply:
What should the UI be?

Having make take another arg, ala `make <schema> <word>`, or a `schema <schema>` command?

(My vote goes for the latter)

Reply:
frankly this file just serves to bring schemas in Anki into the helper
so creating another function `word_to_ChineseRecognitionOnlyWithStroke` is fine

Reply:
no but I mean how should the user tell the program whether they want stroke or not

Reply:
oh yeah, `schema <schema>` makes sense to me (with the default one being `WithStroke`), but tbh you don't have to implement that in this PR
just changing the used schema directly is fine right now